src/database/seeding/seeding.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { SeedingService } from './seeding.service';
+import { SeedingController } from './seeding.controller';
+
+@Module({
```

Comment:
Again, you don't need a full module for seeding data. Seeding should only happen on startup when the database is empty.